### PR TITLE
Ansible sanity - move lookup.py

### PR DESCRIPTION
**Pull request by abikouo**
Depends-On: https://github.com/ansible-collections/vmware.vmware_rest/pull/235

##### SUMMARY

move file in order to be compliant with https://github.com/ansible/ansible/pull/75254#issuecomment-880099069

##### ISSUE TYPE

- Feature Pull Request

### CODE REVIEW

File: plugins/lookup/turbo_demo.py [hunk 1]
```diff
@@ -1,8 +1,6 @@
 # Copyright: (c) 2021, Aubin Bikouo (@abikouo)
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
-from __future__ import absolute_import, division, print_function
-
 __metaclass__ = type
```
Comment (goneri) [inline plugins/lookup/turbo_demo.py:4]: You can also drop the `__metaclass__ = type`. It's Py2 specific.

**Comment by goneri** [comment]
Thank you Aubin!

**Comment by abikouo** [comment]
recheck

**Comment by abikouo** [comment]
recheck

**Comment by abikouo** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Pull request merged**